app/backend/src/clj/teet/cooperation/cooperation_commands.clj

```diff
@@ -203,3 +221,13 @@
                 [:db/retractEntity contact-id]]
                (throw (ex-info "Application has no contact info"
                                {:error :no-contact-info})))})
+
+(defcommand :cooperation/delete-third-party
+  {:doc "Delete a third party. It must not have any applications."
+   :spec (s/keys :req [:db/id])
+   :context {:keys [user db]}
+   :payload {id :db/id}
+   :project-id (cooperation-db/third-party-project-id db id)
+   :authorization {:cooperation/edit-3rd-party {:entity-id id}}
+   :pre [(empty? (:cooperation.3rd-party/applications (du/entity db id)))]
```

Comment:
So you can't delete third parties even if all the applications are marked as deleted?

Reply:
oh yes, obviously it should be possible then